tests/scripts/thread-cert/message.py

```diff
@@ -166,6 +166,18 @@ def assertMleMessageContainsTlv(self, tlv_class_type):
                 break
 
         assert(contains_tlv == True)
+        return tlv
+
+    #To confirm if Leader contains the Route64 TLV with router_quantity assigned Router IDs.
```

Comment:
would it be better to update the comment as below?
> Verify the number of the active routers

Reply:
fixed